crates/proto/proto/responses.proto

```diff
@@ -47,7 +53,7 @@ message SyncStateResponse {
 
     // List of IDs for transactions executed against requested accounts between
     // `block_num + 1` and `block_header.block_num`
-    repeated digest.Digest transactions = 6;
+    repeated TransactionUpdate transactions = 6;
```

Comment:
To make things more clear - should we say `request.block_num + 1` and `self.block_header.block_num`? Otherwise, I'm not sure it is clear what `block_num` refers to.

Also, similar comment for other doc comments for this message.

Reply:
yep - `request` / `response` works just fine.

Reply:
changed to request/response